quickwit/quickwit-query/src/tokenizers/mod.rs

```diff
@@ -128,6 +132,69 @@ pub fn get_quickwit_fastfield_normalizer_manager() -> &'static TokenizerManager
     &QUICKWIT_FAST_FIELD_NORMALIZER_MANAGER
 }
 
+#[derive(Debug, Clone)]
+struct EmitEmptyTokenizer<T>(T);
```

Comment:
We need a comment to express the intent on that kind of tokenizer. It is very difficult for a future readers to infer the point of this.

Reply:
This tokenizer emits the empty token whenever the underlying tokenizer emits no token.
I am not sure this is the behavior we want.

For instance, a stop word would result in the emission of the empty token.

A stricter behavior, emitting an empty token iff the text is effectively empty might actually make more sense. I suspect the code would be simpler too.